src/ripple/app/paths/PathState.h

```diff
@@ -35,20 +36,20 @@ class PathState : public CountedObject <PathState>
     using Ptr = std::shared_ptr<PathState>;
     using List = std::vector<Ptr>;
 
-    PathState (STAmount const& saSend, STAmount const& saSendMax)
+    PathState (PaymentView& parent,
+            STAmount const& saSend,
+                STAmount const& saSendMax)
         : mIndex (0)
         , uQuality (0)
         , saInReq (saSendMax)
         , saOutReq (saSend)
     {
+        view_.emplace(parent);
```

Comment:
I'm looking closely at this and I'm not understanding the difference. In both cases we call the `PaymentView(PaymentView&)` constructor, there's no variables being moved or parameter packs. I'm missing something.